### PR TITLE
DEVDOCS-5904: [Update] fix issues in orders.v3.yml

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5904]

## What changed?
Fix incorrect information in order file

## Release notes draft

Bug fix

* 

## Anything else?

ping {names}

[DEVDOCS-5904]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5904?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -661,13 +648,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/MetafieldResponse'
-        '404':
```
Comment (6juara9) [inline reference/orders.v3.yml:664]: I guess this one is possible to get, I'm just not sure about the response body :)

**Review by 6juara9 — COMMENTED** [review]
- Can we remove all 500's?
- Can we apply these changes to all metafields (not only order metafields)?

File: reference/orders.v3.yml (hunk 1 above)
Comment (bc-traciporter) [reply reference/orders.v3.yml:664]: @6juara9 I updated the response body.
Screenshot 2024-05-03 at 11 44 04 AM

Comment (kskomro) [reply reference/orders.v3.yml:664]: also not sure if we want it to look like this:
Screenshot 2024-05-06 at 14 00 19
The text is on response code description not title part of body. Wdyt?

Comment (bc-traciporter) [reply reference/orders.v3.yml:664]: Hi @kskomro My preview looks like the following:
Screenshot 2024-05-06 at 9 19 56 AM

**Review by kskomro — APPROVED** [review]

**Review by bc-andreadao — APPROVED** [review]

**Review by bc-andreadao — APPROVED** [review]

**Pull request merged**